xclim/indices/helpers.py

```diff
@@ -401,3 +403,53 @@ def wind_speed_height_conversion(
             return ua * np.log(67.8 * h_target - 5.42) / np.log(67.8 * h_source - 5.42)
     else:
         raise NotImplementedError(f"'{method}' method is not implemented.")
+
+
+def gather_lat(da: xr.DataArray) -> xr.DataArray:
+    """Gather latitude coordinate using cf-xarray.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        CF-conformant DataArray with a "latitude" coordinate.
+
+    Returns
+    -------
+    xarray.DataArray
+        Latitude coordinate.
+    """
+    try:
+        lat = da.cf["latitude"]
+        return lat
+    except KeyError as err:
+        n_func = stack()[1].function
+        msg = (
+            f"{n_func} could not find latitude coordinate in DataArray. "
+            "Try passing it explicitly (`lat=ds.lat`)."
+        )
+        raise ValueError(msg) from err
+
+
+def gather_lon(da: xr.DataArray) -> xr.DataArray:
+    """Gather longitude coordinate using cf-xarray.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        CF-conformant DataArray with a "longitude" coordinate.
+
+    Returns
+    -------
+    xarray.DataArray
+        Longitude coordinate.
+    """
+    try:
+        lat = da.cf["longitude"]
+        return lat
+    except KeyError as err:
+        n_func = stack()[1].function
```

Comment:
If I was to use this `gather_lon` in a script would the error message work?

If we want to restrict the use of these two new functions, I would suggest making them private, ex: `_gather_lon`.

Reply:
Good point. Will address.

Reply:
Done.